packages/stacked_tools/lib/src/services/template_service.dart

```diff
@@ -339,4 +467,51 @@ class TemplateService {
       '$renderedTemplate\n$modificationIdentifier',
     );
   }
+
+  /// It reverts the modification applied to a file by [modificationTemplate]
+  Future<String> templateWithoutModifiedFileContent({
```

Comment:
No unit tests has been added for this functionality. Please add tests to confirm that all this works objectively.

Reply:
I've added 24 tests, they are organised as followed:

4 tests for single line templates
4 tests for multi-line templates, which also change on running dart or flutter format on them, for ex:
   ```
int getSomeInt(){
return 1;
}
```
This on getting formatted becomes:
```
int getSomeInt () {
  return 1;
}
```
and finally 4 for multi-line templates that do not get changed even if you run dart or flutter format on them.

12 tests where the template doesn't use any variable like {{serviceName}}.
12 tests where the template does use variables like {{serviceName}}.

Both types have the same distribution as I mentioned above, i.e. 4 single line, 4 multi-line (change on getting formatted), and 4 multi-line (don't change on getting formatted)